### ISSUE TITLE
Can we have latest tag on quay.io/eclipse/che-php-7 & quay.io/eclipse/che-nodejs10-community

### ISSUE THREAD

**Issue by SDAdham** [eclipse-che/che#17301]
The images mentioned in the title doesn't seem to have tag latest, can you please add them?

**Comment by benoitf** [comment]
Hi @SDAdham,

I'm wondering why you would require a latest tag on these images ?
they're tagged on each Eclipse Che version, so it may not work at all using `latest`

Maybe you're just interested in the `nightly` tag ?

**Comment by SDAdham** [comment]
Because every time I update che, I have to go into every devfile and update the tag to the latest version.

Since they are already following che's version, why not just add a `latest` tag?

Thanks @benoitf

**Comment by SDAdham** [comment]
I created a quick extension https://github.com/SDAdham/vs-code-che for vs code, that would get current workspace and also would show/hide `Che`'s navbar on the left. I didn't test it yet on Eclipse che. I want to rebuild `code-server` image with RUN command `RUN code-server --install-extension /tmp/vs-code-che-0.0.1.vsix`

But not sure how can I get `che` to load my image if I publish the image, from what I noticed in the devfile, if I add image, it would say `Error. Property image is not allowed.`
